tests/templates/kuttl/ldap/install-openldap.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: secrets.stackable.tech/v1alpha1
+kind: SecretClass
+metadata:
+  name: openldap-tls
+  namespace: $NAMESPACE
```

Comment:
I like the overall direction :+1: 
Please keep in mind that SecretClasses are AFAIK ClusterScope, so you need to create a unique name for every test suite and step

Reply:
Yeah removed that namespace for SecretClass again